tests/test_main.py

```diff
@@ -0,0 +1,109 @@
+import pytest
+import os
+from gha_runner.__main__ import (
+    parse_aws_params,
+    env_parse_helper,
+)
+
+
+def test_env_parse_helper():
+    base: dict[str, str] = {}
+    assert env_parse_helper(base, "TEST_ENV_VAR_GHA_RUNNER", "test") == base
```

Comment:
Python `dict` is mutable, and `env_parse_helper` mutates the input dict, as opposed to returning a new one. So currently this test would always be true (in fact, you could test with `is` and you'd get `True`).

```suggestion
    assert env_parse_helper(base, "TEST_ENV_VAR_GHA_RUNNER", "test") == {}
```